source/documentation/obligationsandreturns.html.md

```diff
@@ -201,93 +295,44 @@ The [Loss claims](https://developer.service.hmrc.gov.uk/api-documentation/docs/a
 
-If a customer needs to report information to HMRC that is not yet supported under MTD, they will need to complete a [Self Assessment tax return](https://www.gov.uk/self-assessment-tax-returns).  To carry back a loss the customer should contact HMRC.  HMRC will apply this manually.
+## Crystallisation 
 
-## When the customer is ready to crystallise
+The software will have to let HMRC know that the customer is ready to crystallise. To do this you must  call the Intent to crystallise endpoint. This will start the crystallisation process in HMRC. It will trigger the business validation rules (which will become errors rather than warnings) and generate a final liability calculation.
 
-The software will have to let HMRC know that the customer is ready to crystallise. You can do this by calling the [intent to crystallise](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/self-assessment-api/2.0#crystallisation_intent-to-crystallise_post_accordion) endpoint. This will start the crystallisation process and trigger the business validation rules and generate a final liability calculation.
+The intent to crystallise response includes a calculationId the same as the trigger calculation endpoint. Software will then have to retrieve the calculation using calculationId  to retrieve a tax calculation endpoint to get the calculation output.
 
-The intent to crystallise response includes a Calculation ID similar to the trigger calculation endpoint. Software will then have to retrieve the calculation using the ‘Retrieve a Tax Calculation’ endpoint to get the calculation output.
+NEED TO ADD INFO HERE ABOUT THE TAX CALC FOR CRYSTALLISATION AS MENTIONED ABOVE IN TAX CALC SECTION 
 
 The Calculation output provides a summary of each income source (for example self-employment, UK Property, UK bank and building society interest), plus a breakdown of allowances and reliefs applied, and a breakdown of the income tax and NIC payable.) broadly the equivalent of the current SA302.
 
-Once software has called the intent to crystallise endpoint, this will trigger a final liability calculation and software will receive a Calculation ID. 
+Once software has called the intent to crystallise endpoint, this will trigger a final liability calculation and software will receive a Calculation ID.
 
-The software must retrieve the intent to crystallise calculationId and display the calculation to the customer.  The customer must review this information and confirm it is complete and correct by sending the declaration.
+The software must us the  intent to crystallise calculationId to retrieve the final calculation and display that calculation to the customer. The customer must review this calculation  and confirm it is complete and correct by sending the declaration.
```

Comment:
"The software must _us_ the  intent to crystallise" -> "The software must _use_ the intent to crystallise"